packages/libraries/core/src/client/agent.ts

```diff
@@ -56,6 +56,7 @@ export interface AgentOptions {
   maxRetries?: number;
   /**
    * 200 by default
+   * @deprecated This is no longer used.
```

Comment:
nit: Do we have an alternative field or something related? (if so, worth documenting it)